src/components/shared/RadioField/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+type RadioFieldProps = {
+  id: string;
+  label: string;
+  name: string;
+  onBlur: () => void;
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  value: string;
+};
+
+const RadioField = ({
```

Comment:
Similarly, we could make a component for a list of radio buttons

Reply:
Same reasoning as the `DropdownField`.